rmf_fleet_adapter/src/rmf_fleet_adapter/TaskManager.cpp

```diff
@@ -70,6 +90,7 @@ void TaskManager::queue_task(std::shared_ptr<Task> task, Start expected_finish)
   {
     rmf_task_msgs::msg::TaskSummary msg;
     msg.task_id = _queue.back()->id();
+    msg.task_profile.task_id = _queue.back()->id();
```

Comment:
Populate `robot_name here`, similar to other lines. this will be helpful in showing the allocation.
```
msg.robot_name = this->_context->name();
```

Reply:
https://github.com/osrf/rmf_core/pull/199/commits/45ceb289e27a8ccbda6525436da737ef9737a6cc